kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -32,6 +32,10 @@ class eZUser extends eZPersistentObject
     /// Passwords in PHP default format
     const PASSWORD_HASH_PHP_DEFAULT = 7;
 
+    /// Default password hashing algorithm, used in case of invalid configuration or usage.
+    /// Update this if support for better algorithms is added.
+    const DEFAULT_PASSWORD_HASH = self::PASSWORD_HASH_MD5_USER;
```

Comment:
Why not `PASSWORD_HASH_BCRYPT` ?

Reply:
@emodric Artifact from earlier iteration. Should be `PASSWORD_HASH_PHP_DEFAULT`. I'll correct it. Thanks!